save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityHeader.kt

```diff
@@ -247,6 +249,18 @@ internal val headerMenu: FC<HeaderMenuProps> = FC { props ->
                                 props.setIsTableView { !it }
                             }
                         }
+                        if (props.selectedMenu == VulnerabilityTab.INFO) {
+                            a {
```

Comment:
I think ```Link { }``` would be preferable here

Reply:
looks like `Link` is a link to some page on site.
I tried it and it redirect to main page and that all

Reply:
Oh, I didn’t notice that you have a file download here. OK

Reply:
found example with full url, will try to again

Reply:
tried with `Link {}` and relative path -- doesn't work too